libraries/core-react/src/Divider/Divider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { divider as tokens } from './Divider.tokens'
+
+type StyleProps = {
+  backgroundColor: string
+  marginTop: string
+  marginBottom: string
+  dividerHeight: string
+}
+
+const StyledDivider = styled.hr<StyleProps>`
+  border: none;
+  background-color: ${(props) => props.backgroundColor};
+  margin-top: ${(props) => props.marginTop};
+  margin-bottom: ${(props) => props.marginBottom};
+  height: ${(props) => props.dividerHeight};
+`
+
+type Props = {
+  /** Color variants
+ @default medium */
+  color?: 'lighter' | 'light' | 'medium'
+  /** Vertical spacings 
+   @default medium */
+  variant?: 'small' | 'medium'
+  className?: string
```

Comment:
Should we ignore all classnames? Or add something about the possibility of overriding the default styles 🤔

Reply:
I don't know. I don't have any strong opinions here 🤷‍♀️